src/libpmemset/source.c

```diff
@@ -138,9 +175,18 @@ pmemset_source_from_fileU(struct pmemset_source **src, const char *file)
 		Free(srcp);
 		return PMEMSET_E_ERRNO;
 	}
+
+	ret = pmemset_source_create(srcp);
```

Comment:
why do we have two copies of almost exactly the same function? `pmemset_source_from_fileU` and `pmemset_source_from_file`.